src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1437,4 +1453,26 @@ I agree to all of the foregoing:</value>
   <data name="Razor_CSharp_Language_Server_Client" xml:space="preserve">
     <value>Razor C# Language Server Client</value>
   </data>
+  <data name="Type_name_does_not_parse_correctly" xml:space="preserve">
+    <value>Type name does not parse correctly</value>
```

Comment:
```suggestion
    <value>Type name has a syntax error</value>
```

Since I think "syntax error" might be a more common term?

Reply:
Took this one, thanks.